packages/block-library/src/navigation/edit.js

```diff
@@ -84,7 +77,7 @@ function Navigation( {
 			className: 'wp-block-navigation__container',
 		},
 		{
-			allowedBlocks: ALLOWED_BLOCKS,
+			allowedBlocks,
```

Comment:
If it's specified in the block.json, my feeling is a block developer shouldn't have to add some logic to the block edit function. Is there a way to make it work without it having to pass it through as a prop?

Reply:
I imagine block.json could override the prop, with the prop deprecated.